drive/carts/vkeyboard.p8

```diff
@@ -62,7 +62,28 @@ function Keyboard:new(key_color, border_color, selected_color, x, y, confirm_fn)
     setmetatable(obj, self)
     return obj
 end
+--invert case
+function Keyboard:corrected_text()
```

Comment:
was thinking it made more sense to implement it on the rust side, but this is fine too